packages/cli/generators/repository/templates/src/repositories/repository-crud-default-template.ts.ejs

```diff
@@ -1,14 +1,14 @@
 import {<%= repositoryTypeClass %>, juggler} from '@loopback/repository';
 import {<%= modelName %>} from '../models';
-import {<%= dataSourceClassName %>} from '../datasources';
+import {<%= dataSourceName %>DataSource} from '../datasources';
```

Comment:
I don't think it's a good idea to concatenate strings in the EJS template. We should calculate the `dataSourceClassName` in the params passed to the EJS.

If we want to have both `dataSourceName` and `dataSourceClassName`, that's perfectly fine.

Reply:
Ok. I will create dataSourceClassName.